modules/arm_plugin/Dockerfile.RPi32

```diff
@@ -0,0 +1,137 @@
+        curl \
+        nano \
+        tar \
+        bzip2 \
+        unzip \
+        cpio \
+        cifs-utils \
+        locales \
+        rsync \
+        apt-transport-https \
+        debian-archive-keyring \
+        sudo \
+        openssh-client \
+        default-jre \
+        p7zip-full \
+        software-properties-common \
+        dirmngr \
+        gnupg \
+        netcat-openbsd \
+        dh-autoreconf \
+        libcurl4-gnutls-dev \
+        libexpat1-dev \
+        gettext \
+        libz-dev \
+        libssl-dev \
+        build-essential \
+    && \
+    locale-gen en_US.UTF-8 && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN dpkg --add-architecture armhf && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends -f -o Dpkg::Options::="--force-confnew" \
+        build-essential \
+        libusb-1.0-0-dev:armhf \
+        software-properties-common \
+        crossbuild-essential-armhf \
+        zlib1g-dev \
+        libffi-dev \
+        libssl-dev \
+        chrpath \
+        libssl-dev \
+        libprotobuf-dev \
+        libprotoc-dev \
+        protobuf-compiler \
+        # For ARM CPU plugin
+        scons
+
+RUN wget https://www.cmake.org/files/v3.13/cmake-3.13.3.tar.gz && \
+    tar xf cmake-3.13.3.tar.gz && \
+    (cd cmake-3.13.3 && ./bootstrap --parallel=$(nproc --all) && make --jobs=$(nproc --all) && make install) && \
+    rm -rf cmake-3.13.3 cmake-3.13.3.tar.gz
+
+RUN curl -sSf "https://packagecloud.io/install/repositories/github/git-lfs/config_file.list?os=debian&dist=stretch&source=script" > "/etc/apt/sources.list.d/github_git-lfs.list"
+RUN curl -L "https://packagecloud.io/github/git-lfs/gpgkey" 2> /dev/null | apt-key add - &>/dev/null && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends  --allow-unauthenticated \
+        git-lfs \
+        && \
+    rm -rf /var/lib/apt/lists/*
+
+# build and install git 2.22
+RUN wget https://github.com/git/git/archive/v2.22.2.tar.gz && \
+    tar xf v2.22.2.tar.gz && \
+    cd git-2.22.2 && \
+    make configure && \
+    ./configure --prefix=/usr && \
+    make all && \
+    make install && \
+    rm -rf /git-2.22.2 && \
+    rm /v2.22.2.tar.gz
+
+#configure paths
+RUN mkdir -p /armcpu_plugin/armcpu_debian32_rpi
+WORKDIR /armcpu_plugin
+ENV DEV_HOME /armcpu_plugin
+ENV OPENCV_HOME=$DEV_HOME/opencv
+ENV OPENVINO_HOME $DEV_HOME/openvino
+ENV OPENVINO_CONTRIB $DEV_HOME/openvino_contrib
+ENV ARM_PLUGIN_HOME $OPENVINO_CONTRIB/modules/arm_plugin
+ENV STAGING_DIR $DEV_HOME/armcpu_debian32_rpi
+
+ENV SOURCE_UPDATES_OF 20210120_11
+
+#OpenCV
+RUN git clone https://github.com/opencv/opencv.git $OPENCV_HOME && \
+    mkdir $OPENCV_HOME/build && \
+    cd $OPENCV_HOME/build && \
+    cmake .. -DCMAKE_BUILD_TYPE=Release -DBUILD_LIST=imgcodecs,videoio,highgui \
+             -DCMAKE_TOOLCHAIN_FILE=$OPENCV_HOME/platforms/linux/arm-gnueabi.toolchain.cmake \
+             -DCMAKE_STAGING_PREFIX=$STAGING_DIR && \
+    make -j32 && \
```

Comment:
> -j32

This is bad practice to oversubscribe resources.
Use `$(nproc)` for that or use Docker's `--build-arg` with some default (e.g. 4).

Reply:
set `make -j$(nproc)` in `Dockerfile.RPi32` and `Dockerfile.RPi63`